core/src/test/java/com/linecorp/armeria/server/WrappingTransientRpcServiceTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import com.linecorp.armeria.common.RpcResponse;
+
+class WrappingTransientRpcServiceTest {
+
+    static final RpcService FOO = (ctx, req) -> RpcResponse.of("foo");
```

Comment:
nit: A `Service` is not constant and `FOO` is not good for a service name. Just use `fooService`?